docs/content/introduction/productvisie.md

```diff
@@ -66,38 +73,61 @@ Bij de start van dit traject hanteren we de volgende uitgangspunten:
 
+- Eisen rond [Duurzame toegankelijkheid](https://wiki.nationaalarchief.nl/pagina/DUTO:Kwaliteitseisen) worden vanaf het begin in de standaarden verwerkt.
+
+- Archivering speelt een rol bij de uitvoering van elk proces, vanaf de start tot aan [overbrenging](https://wiki.nationaalarchief.nl/pagina/DUTO:Overbrenging) of [vernietiging](https://wiki.nationaalarchief.nl/pagina/DUTO:Vernietigen). De gegevensstandaard voor archiefstukken [TMLO](https://archief2020.nl/nieuws/toepassingsprofiel-metadatering-lokale-overheden)is al geïntegreerd in RGBZ en ImZTC.
+
 
 ## Realisatie
 
 De realisatie gaat van start zonder complete blauwdruk van wat gebouwd moet worden. Door user-stories zo goed mogelijk in te vullen en vanuit oogpunt architectuur in de gaten te houden dat designchoices worden gemaakt die ruimte openlaten voor verdere ontwikkeling in de richting van de visie, komt de nieuwe standaard stukje bij beetje tot stand.
 
-Ontwikkeling valt uiteen in 3 delen die niet los van elkaar gezien kunnen worden:
-- Opstellen en realiseren van een nieuwe ZDS 2.0 API
-- Beschikbaar maken van een Open Source referentieimplementatie
-- Realiseren van toepassingen voor burgers of gemeenten die gebruik maken van zaakgegevens uit het zakenregistratiecomponent, via de gestandaardiseerde API
-
 Om een snelle start te maken wordt gestart met alle gemeenten en partijen waarvan de interesse bekend is. Aanhakers zijn nadrukkelijk welkom.
 
-De ontwikkeling gebeurt geheel Open Source, in de GitHub repository [VNG-Realisatie/gemma-zaken](https://github.com/VNG-Realisatie/gemma-zaken).
+De ontwikkeling gebeurt in de GitHub repository [VNG-Realisatie/gemma-zaken](https://github.com/VNG-Realisatie/gemma-zaken).
 
-Ook de backlog wordt publiek bijgehouden, samengesteld uit GitHub issues op dit open [Kanban bord](https://github.com/VNG-Realisatie/gemma-zaken/projects/1).
+Ook de backlog wordt publiek bijgehouden, samengesteld uit GitHub issues op dit open [Scrum bord](https://github.com/VNG-Realisatie/gemma-zaken/projects/1).
 
-Bij de ontwikkeling van de API's wordt gestreefd naar backwards compatibility. 
+Bij de ontwikkeling van de API's wordt gestreefd naar backwards compatibility.
 
 Voor meer informatie over de samenwerking, zie [samenwerking.md](./samenwerking.md)
 
-Het open source zakenregistratiecomponent is een [referentie implementatie](https://nl.wikipedia.org/wiki/Referentie-implementatie) van de (nieuw te ontwikkelen) specificatie. De referentie implementatie heeft de volgende karakteristieken:
+Naast de (nieuw te ontwikkelen) OpenAPI 3 specificatie wordt een [referentie implementatie](https://nl.wikipedia.org/wiki/Referentie-implementatie) gemaakt. De referentie implementatie heeft de volgende karakteristieken:
 
 * Wordt gelijktijdig ontwikkeld met de specificatie en (geautomatiseerde) test suite;
+* Maakt het mogelijk ketentesten uit te voeren, wat vraagt om persistentie van testdata;
```

Comment:
Voor de referentie-implementatie betekent dit dat geen dummy-berichten kunnen worden uitgewisseld, maar dat persistente data in achterliggende databases noodzakelijk is. Dit wijkt af van wat bijv. gebeurt in het StUF-testplatform. Het voordeel is dat iedereen ketentesten kan uitvoeren tegen de referentieimplementatie.